str/str_iterative_gangstr_runner.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+# pylint: disable=import-error,duplicate-code
+
+"""
+This script uses GangSTRv2.5 to call STRs on WGS cram files.
+Required input: --variant-catalog (file path to variant catalog), --dataset, and external sample IDs
+For example:
+analysis-runner --access-level test --dataset tob-wgs --description 'tester' --output-dir 'tester' str_iterative_gangstr_runner.py --variant-catalog=gs://cpg-tob-wgs-test/hoptan-str/Illuminavariant_catalog.json --dataset=tob-wgs TOB1XXXX TOB1XXXX
+
+Required packages: sample-metadata, hail, click, os
+pip install sample-metadata hail click
+
+"""
+import os
+import logging
+
+import click
+import hailtop.batch as hb
+
+from sample_metadata.model.analysis_type import AnalysisType
+from sample_metadata.model.analysis_query_model import AnalysisQueryModel
+from sample_metadata.apis import AnalysisApi, SampleApi
+from sample_metadata.models import AnalysisStatus
+
+from cpg_utils.config import get_config
+from cpg_utils.hail_batch import remote_tmpdir, output_path, reference_path
+
+config = get_config()
+
+SAMTOOLS_IMAGE = config['images']['samtools']
+GANGSTR_IMAGE = config['images']['gangstr']
+
+
+# inputs:
+# variant catalog
+@click.option('--variant-catalog', help='Full path to Illumina Variants catalog')
+# input project ID
+@click.option('--dataset', help='dataset to operate on, eg: tob-wgs')
```

Comment:
Optional: You could make the default here the default from the config you had before. So running the script without a dataset param automatically uses the dataset provided in the config?